docs/node/workflow-external-dependencies.md

```diff
@@ -0,0 +1,57 @@
+# External Dependencies
```

Comment:
i like the idea of calling this "advanced" section to make clear it is optional. also we should put all these `workflow-*` docs into one page, to be inline with the other 3 SDK's

Reply:
Agreed, I want to reorganize these pages at soon.

Reply:
For now I put all of the workflow related pages in a `Workflows` category.